src/stateless/slider.vue

```diff
@@ -0,0 +1,597 @@
+        },
+        inputType () {
+            return this.decimalPlacesCount === 0 ? 'number' : 'float'
+        }
+    },
+    methods: {
+        startDrag (e) {
+            if (this.disabled) return
+
+            this.bounds = this.$refs.slider.getBoundingClientRect() // hotfix, will try to find a better way
+            this.handle = this.$refs.handle.getBoundingClientRect() // hotfix, will try to find a better way
+
+            this.changedFlag = true
+            this.draggingFlag = true
+            this.inputFlag = false
+
+            this.setPosition(e)
+            window.addEventListener('mouseup', this.stopDrag)
+            window.addEventListener('mousemove', this.setPosition)
+            this.$refs.slider.focus()
+        },
+        setPosition (e) {
+            let percent = (e.clientX - this.bounds.x) / this.bounds.width
+            percent = Math.min(Math.max(0, percent), 0.999999)
+
+            let edgeStepOffset = this.stepPercentage / 2
+            this.index = Math.floor((percent - edgeStepOffset) / this.stepPercentage) + 1
+
+            e.preventDefault()
+        },
+        stopDrag () {
+            this.draggingFlag = false
+            window.removeEventListener('mouseup', this.stopDrag)
+            window.removeEventListener('mousemove', this.setPosition)
+            this.$emit('value', this.currentValue)
+        },
+        keyboard (e) {
+            if (this.disabled) return
+            this.changedFlag = true
+
+            let key = e.keyCode
+
+            if ((key === 38 || key === 39) && this.index < this.stepsCount)
+                this.index++
+            else if ((key === 37 || key === 40) && this.index > 0)
+                this.index--
+
+            this.$emit('value', this.currentValue)
+            e.preventDefault()
+        },
+        handleInput (value) {
+            let numberValue = parseFloat(value)
+
+            this.inputFlag = true
+            this.inputValue = value
+            
+            if (this.min <= numberValue && numberValue <= this.max)
+                this.index = (value - this.min) / this.step
+        },
+        isUnitActive (index) {
+            return index / 19 <= this.percentage
+        },
+        isValidInput (value, onError) {
+            return (value < this.min || this.max < value) ? '' : null
+        },
+        tickClass (index) {
+            let hidden = false
+            if (this.domReady) {
+                let position = index / 19 * this.bounds.width
+                hidden = position <= this.minThreshold || position >= this.maxThreshold
+            }
+
+            return {
+                'slider-label__unit--active': this.isUnitActive(index),
+                'slider-label__unit--hidden': hidden
+            }
+        },
+        labelClass (index) {
+            return {
+                'slider-label__unit--active': this.isUnitActive(index) && this.size === 'normal'
```

Comment:
You can use `prefix` filter and return only modifier name here (`active: this.isUnitActive(index) && this.size === 'normal'`), then in template `labelClass(0) | prefix('slider-label__unit--')`